src/CrossmintEmbed.ts

```diff
@@ -140,6 +173,8 @@ export default class CrossmintEmbed {
         });
     }
 
+    // When an AA is selected, just that wallet can be connected.
+    // So, we don't need to adapt signMessages to support AA wallets
```

Comment:
this is a good note for the PR, but here we should say
```suggestion
    /// Does not support AA signatures because currently, if an AA wallet is connected, we only allow one wallet
```